fe-react/files/metadata.yml

```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: React
+  description: "React is a JavaScript library for building user interfaces. It is maintained by Facebook and a community of individual developers and companies. Technologies: react unpinned"
```

Comment:
@georgfedermann can you double-check? I thought I had found a 16.4.

Reply:
I only found this in the Jenkinsfile:
`npm init react-app ${componentId}`
Thus thought, the react version was unpinned. Where did you find the v16.4?

Reply:
Hmm... `npm init react-app` requires the `create-react-app` [npm package](https://www.npmjs.com/package/create-react-app), but I cannot see where that is installed.